Makefile

```diff
@@ -60,10 +60,13 @@ errcheck:
 	go get -v github.com/kisielk/errcheck
 	errcheck -verbose -blank $(PKGS)
 
-fmt:
+check-fmt:
 	bash -c "diff -u <(echo -n) <(gofmt -l -d -s -e $(GO_FILES))"
 
-pretest: fmt lint vet errcheck simple
+do-fmt:
+	@./hack/do-gofmt.sh $(PKGS)
```

Comment:
$(PKGS) already filters out vendor and generated code, so I'm not sure we need the script

Reply:
removed extra script